plugins/DuskVerb/tools/tuner/baseDelay_sweep.py

```
@@ -0,0 +1,178 @@
+                                     direction="minimize",
+                                     sampler=optuna.samplers.TPESampler(seed=42))
+    else:
+        study = optuna.create_study(direction="minimize",
+                                     sampler=optuna.samplers.TPESampler(seed=42))
+
+    study.optimize(objective, n_trials=args.trials, n_jobs=args.workers)
+
+    best = study.best_trial
+    print("\n========== BEST ==========")
+    print(f"Trial {best.number}  loss={best.value:.4f}")
+    for band, (lo, hi, target) in targets.items():
```

Ruff (0.15.15)

[warning] Loop control variable `lo` not used within loop body

(B007)